Validation/Geometry/src/MaterialBudgetHcalHistos.cc

```diff
@@ -102,9 +102,11 @@ void MaterialBudgetHcalHistos::fillStartTrack(const G4Track* aTrack) {
     intLength_.clear();
   }
 
-  edm::LogVerbatim("MaterialBudget") << "MaterialBudgetHcalHistos: Track " << aTrack->GetTrackID() << " Code " << theID
-                                     << " Energy " << theEnergy / CLHEP::GeV << " GeV; Eta " << eta_ << " Phi "
-                                     << phi_ / CLHEP::deg << " PT " << dir.perp() / CLHEP::GeV << " GeV *****";
+  if ((std::abs(eta_) >= etaMinP_) && (std::abs(eta_) <= etaMaxP_))
+    edm::LogVerbatim("MaterialBudget") << "MaterialBudgetHcalHistos: Track " << aTrack->GetTrackID() << " Code "
+                                       << theID << " Energy " << theEnergy / CLHEP::GeV << " GeV; Eta " << eta_
```

Comment:
Unless this PR is particularly urgent, now is the time to remove CLHEP units from this file. Replace `#include "CLHEP/Units/GlobalSystemOfUnits.h"` with `#include "DataFormats/Math/interface/GeantUnits.h"`. You can add `using geant_units::operators` for convenience.
`theEnergy / CLHEP::GeV` should be replaced with `convertsUnitTo(1._GeV, theEnergy)` and
`/ CLHEP::deg` should be replaced with `convertRadToDeg()`.